compiler-rt/test/tysan/violation-pr47137.c

```diff
@@ -4,6 +4,7 @@
 // https://github.com/llvm/llvm-project/issues/47137
 #include <stdio.h>
 #include <stdlib.h>
+#include <stdint.h>
```

Comment:
This line seems unrelated to the other changes, any reason for it?

Reply:
The test uses uint64_t, which is in stdint.h. When I was running these tests, not having that file included gave me a compile error.